dallinger/config.py

```diff
@@ -72,12 +72,18 @@ def is_valid_json(value):
     ("logfile", six.text_type, []),
     ("loglevel", int, []),
     ("mode", six.text_type, []),
+    ("mturk_qualification_blocklist", six.text_type, ["qualification_blacklist"]),
+    ("mturk_qualification_requirements", six.text_type, [], False, [is_valid_json]),
     ("num_dynos_web", int, []),
     ("num_dynos_worker", int, []),
     ("organization_name", six.text_type, []),
     ("port", int, ["PORT"]),
-    ("mturk_qualification_blocklist", six.text_type, ["qualification_blacklist"]),
-    ("mturk_qualification_requirements", six.text_type, [], False, [is_valid_json]),
+    ("prolific_api_token", six.text_type, [], True),
```

Comment:
I wonder if it at some point we might consider doing this sort of thing as a separate package from Dallinger core, so that it could operate on its own release schedule, etc.

I think we've added enough extensibility to Dallinger for a 3rd party recruiter to register itself, but it would be interesting to find out if that's really the case. Doing this inside of Dallinger core is definitely the path of least resistance though.